apps/ocpp/call_handlers/results/__init__.py

```
@@ -0,0 +1,185 @@
+    "handle_set_monitoring_level_result",
+    "handle_set_network_profile_result",
+    "handle_install_certificate_result",
+    "handle_delete_certificate_result",
+    "handle_certificate_signed_result",
+    "handle_get_installed_certificate_ids_result",
+    "handle_get_variables_result",
+    "handle_set_variables_result",
+    "handle_set_variable_monitoring_result",
+    "handle_clear_variable_monitoring_result",
+    "handle_get_monitoring_report_result",
+]
```

Ruff (0.14.14)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)